db/migrate/20220427191552_create_posts.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.references :user, null: false, foreign_key: true
+      t.string :Title
+      t.string :Text
+      t.integer :Comments_Counters
+      t.integer :Likes_Counter
```

Comment:
Kindly use `snake_case` as this is part of Ruby's best practice 👍 

![image](https://user-images.githubusercontent.com/66112760/165623594-7efe2bb8-be50-4abe-be82-9e4ca374cd52.png)